src/ButtonToggle.js

```diff
@@ -15,72 +15,52 @@ const defaultProps = {
   defaultValue: false,
```

Comment:
To avoid checking for `onClick`, `onBlur`, and `onFocus` you can add defaultProps to the component:
```
const noop = () => {}
const defaultProps = {
   ...,
   onBlur: noop,
   onFocus: noop,
   ...
}

Reply:
Although for 1:1 translation this works as the old implementation did this.